data_inconsistencies.py

```diff
@@ -32,17 +32,35 @@ def get_inconsistencies():
     return incons
 
 
+def print_statistics(file_name, n_incons, firefox, chrome):
+    f = utils.get_all_images()
+    total_img = len(f)
+    incons_f = [int(x) for x in firefox]
+    incons_c = [int(x) for x in chrome]
+    print("Number of Photos: {} " .format(total_img))
+    print("Number of Pair of images: {} " .format(int((total_img - n_incons) / 2)))
+    print("Number of Pair of images possible: {} " .format(int((total_img - n_incons) / 2 + n_incons)))
+    print("Percentage of Firefox Inconsistencies: {}  " .format(int(((n_incons - sum(incons_f)) / n_incons) * 100)))
+    print("Percentage of Chrome Inconsistencies: {} " .format(int(((n_incons - sum(incons_c)) / n_incons) * 100)))
+
+
 def main():
     print('[*] Getting inconsistencies in screenshots...')
     incons = get_inconsistencies()
-    print('[*] {} inconsistencies found.'.format(len(incons)))
+    n_incons = len(incons)
+    firefox = []
+    chrome = []
+    print('[*] {} inconsistencies found.'.format(n_incons))
     print('[*] Writing to inconsistencies.csv... ', end='')
     with open('inconsistencies.csv', 'w') as csvfile:
         writer = csv.writer(csvfile, delimiter=',')
         writer.writerow(['WEBCOMPAT-ID', 'SEQUENCE-NO', 'FIREFOX', 'CHROME'])
         for line in incons:
             writer.writerow(line)
+            firefox.append(line[2])
```

Comment:
Do this in the print_statistics function, as it's unrelated to writing the file.

Reply:
That is, pass `incons` to print_statistics